lintcode/two_pointers/521-Remove-Duplicate-Numbers-in-Array/solution.py

```diff
@@ -34,3 +36,27 @@ def deduplication(self, nums):
             i = i + 1
 
         return i + 1
+
+    def deduplication_by_hash(self, nums):
+        """
+        @param nums: an array of integers
+        @return: the number of unique integers
+        """
+        # <Wayne Shih> 21-Sep-2022
+        # - fast ptr moves faster and takes a look to
+        #   find the number not in the isDup yet
+        # - i ptr means there are no duplicates on LHS of i including i
+        if not nums:
+            return 0
+
+        isDup = set()
```

Comment:
```suggestion
        unique_num_set = set()
```

unique_num_set might be better

Reply:
Resolved as suggestion